tap-setup-scripts/src/functions.sh

```diff
@@ -494,14 +494,15 @@ function tapInstallFull {
   done
 
   banner "Checking for ERRORs in all packages"
+  # Add some delay before checking status again
+  sleep 60
```

Comment:
*question*: Why are we waiting here anyway? What are we waiting for?

Are we trying to wait for another reconciliation cycle of the `tap` `pkgi`, so that we can then assert on the state of the packages?

Wouldn't it be more useful to e.g.
- "Kick" the `tap` `pkgi`/`app`
    - sidenote: "kick" in the sense of adding / updating a label or annotation on the resource; I wouldn't want to introduce a dependency on the `kctrl` tool, which would have a literal `kick` subcommand.
- `kubectl wait` for its status / some jsonpath to change

instead of "randomly" waiting for 1 minute?

Reply:
Ideally, this wait is not required. I added it to make sure that the other reconciliation cycle is settled before we move to check again. Might be helpful in some rare-condition.